### DISCUSSION TITLE
Access to community servers

### DISCUSSION THREAD

**Discussion by nurdism** [MrOkiDoki/BattleBit-Community-Server-API#33] [General]
Hello @MrOkiDoki!
Thank you for opening up the [Community-Server-API](https://github.com/MrOkiDoki/BattleBit-Community-Server-API) to the public early! I'm excited to see what can be created in the future!

I am working on an implementation of the community API for JavaScript/TypeScript over here [battlebit.js](https://github.com/battlebitjs/battlebit.js) I'd like to request access to the community servers to test things out and have a working API server when the community servers become available. Let me know if that would be possible, my discord handle is `nurdism` 

Thankyou for your time 👍🏻

**Comment by Mooshua** [comment]
You should contact Oki or Julgers directly over discord.

**Reply by nurdism** [reply]
I've tried, but got no response 😕
